compiler/passes/ResolveScope.cpp

```diff
@@ -1071,13 +1049,19 @@ Symbol* ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
   }
 
   if (symbols.size() == 1) {
-    // modArg must have been set correctly above
     return symbols[0];
   }
   else if (symbols.size() > 1) {
     // potentially start the error process here
     checkConflictingSymbols(symbols, name, context,
                             traversedRenames, renameLocs);
+    if (numFuncs == symbols.size()) {
+      // All options found were functions, but we found them from different
+      // public imports.  That's okay, though, function resolution will handle
+      // determining which one is the best choice.  Arbitrarily return the
+      // first function
+      return symbols[0];
+    }
```

Comment:
Minor:  this check should happen before `checkConflictingSymbols`. i.e. if all the symbols are functions `checkConflictingSymbols` doesn't do anything but just iterate over the list of symbols. I don't think it'd have real impact on performance or anything. It would just feel more natural to me.